README.md

```diff
@@ -61,14 +61,14 @@ You can also try Elyra via docker.
 The command below starts a clean Elyra environment:
 
 ```
-docker run -it -p 8888:8888 elyra/elyra:1.1.0 jupyter lab --debug
+docker run -it -p 8888:8888 elyra/elyra:latest jupyter lab --debug
 ```
```

Comment:
I believe master should be using the `dev` tag which is built automatically by docker hub on updates. Other docs such as the `JupyterHub guide`.

Reply:
Oooooh, I think see. I was under the assumption we would promote the latest stable version Docker image, whereas you are saying it should be the latest development version image? If that's the case I can change the tag from `latest` to `dev` and add a blurb that hopefully helps the user make an informed decision which image would be the right one.

Reply:
So, currently:
`latest` is updated during a release and points to the latest released version
`dev` is auto-built by docker hub and should be updated after commits (time-wise it depends on build queue size)

Also, note that during "releases" the readme does get updated to the release version, which then is available on the tag.

Reply:
I added a list of options along with a link to Docker Hub to address your feedback